tests/compatibility.py

```diff
@@ -134,5 +136,15 @@ def module_doc_string_tab() -> str:
 
     return ""
 
+
+def get_type_expected(field_type: str):
+    if sphinx.version_info >= (6, 1):
+        optional_match = re.findall(r'Optional\[(\w*)\]', field_type)
+        if optional_match is not None and len(optional_match) > 0:
+            return optional_match[0] + ' | None'  # int | None
+    return field_type  # 'Optional[int]'
+
```

Comment:
Could you please provide a simple doc string with an example of input -> output? This would greatly help to understand what this function does. Moreover, the function name itself is rather generic and requires some guessing.

Reply:
done in 1b7df9b